common/window.py

```diff
@@ -54,7 +54,7 @@ def getclick(self):
 if __name__ == "__main__":
   import numpy as np
   win = Window(200, 200, double=True)
-  img = np.zeros((200, 200, 3), np.uint8)
+  img: np.ndarray = np.zeros((200, 200, 3), np.uint8)
```

Comment:
why can't this be inferred? we don't want to do this all over the codebase

Reply:
The `+=` operator on line 60 throws the below error when unsignted int's are used as the dtype (tried uint16, uint32, and uint64). Regular np.int8, np.int16, etc (and floats for that matter) don't have that issue. We could convert the dtype to np.int16 to keep the same number of character support (albeit some extra memory for the negative number space that isn't being used), keep the type specification, or add an ignore statement to line 60. Personal preference would be to change the dypte since this seems like a valid error.

```
$ mypy common/window.py
common/window.py:60: error: Argument 1 to "__iadd__" of "ndarray" has incompatible type "int"; expected "Union[_SupportsArray[dtype[Union[bool_, unsignedinteger[Any]]]], _NestedSequence[_SupportsArray[dtype[Union[bool_, unsignedinteger[Any]]]]], bool, _NestedSequence[bool]]"
```

Relevant CI build:
https://github.com/commaai/openpilot/runs/6251557005?check_suite_focus=true